components/Checkbox/src/index.scss

```diff
@@ -1,118 +1,10 @@
-  border-color: var(--utrecht-checkbox-invalid-border-color, var(--utrecht-form-control-invalid-border-color));
-}
+@import "~@utrecht/components/checkbox/css";
 
-.denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__icon .denhaag-icon {
-  color: var(--utrecht-checkbox-invalid-color);
-}
-
-.denhaag-checkbox:focus-within .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--focus .denhaag-checkbox__icon {
-  outline: var(--denhaag-focus-border);
-}
-
-.denhaag-checkbox:hover .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--hover .denhaag-checkbox__icon,
-.denhaag-form-control-label:hover .denhaag-checkbox__icon {
-  border-width: var(--denhaag-checkbox-hover-border-width);
-  border-color: var(--denhaag-checkbox-hover-border-color);
-}
-
-.denhaag-checkbox .denhaag-checkbox__input:disabled + .denhaag-checkbox__icon {
-  background-color: var(--utrecht-checkbox-disabled-background-color);
-  border-width: var(--utrecht-checkbox-disabled-border-width);
-  border-color: var(--utrecht-checkbox-disabled-border-color);
-  cursor: default;
-}
-
-.denhaag-checkbox.denhaag-checkbox--error:hover .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--error.denhaag-checkbox.denhaag-checkbox--hover .denhaag-checkbox__icon,
-.denhaag-form-control-label:hover .denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__icon {
-  border-color: var(--denhaag-checkbox-hover-invalid-border-color);
-}
-
-.denhaag-checkbox:hover .denhaag-checkbox__input:checked + .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--hover .denhaag-checkbox__input:checked + .denhaag-checkbox__icon,
-.denhaag-form-control-label:hover .denhaag-checkbox__input:checked + .denhaag-checkbox__icon {
-  background-color: var(--denhaag-checkbox-hover-checked-background-color);
-  border-color: var(--denhaag-checkbox-hover-checked-border-color);
-}
-
-.denhaag-checkbox .denhaag-checkbox__input:checked:disabled + .denhaag-checkbox__icon {
-  background-color: var(--denhaag-checkbox-checked-disabled-background-color);
-  border-color: var(--denhaag-checkbox-checked-disabled-border-color);
-}
-
-.denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__input:checked + .denhaag-checkbox__icon {
-  background-color: var(--denhaag-checkbox-checked-invalid-background-color);
-}
+.utrecht-checkbox--custom {
+  border-radius: var(--utrecht-checkbox-border-radius);
+  margin-inline-end: 8px;
 
-.denhaag-checkbox.denhaag-checkbox--error:hover .denhaag-checkbox__input:checked + .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--error.denhaag-checkbox--hover
-  .denhaag-checkbox__input:checked
-  + .denhaag-checkbox__icon,
-.denhaag-form-control-label:hover
-  .denhaag-checkbox.denhaag-checkbox--error
-  .denhaag-checkbox__input:checked
-  + .denhaag-checkbox__icon {
-  background-color: var(--denhaag-checkbox-hover-invalid-background-color);
-  border-color: var(--denhaag-checkbox-hover-invalid-border-color);
+  &.utrecht-checkbox--disabled {
```

Comment:
Moet dit een dubbele selector zijn, of zou het al werken als 'ie na de `@import` komt?

Reply:
Is aangepast, enkele selector was inderdaad voldoende.